R/options.R

```diff
@@ -32,10 +35,10 @@
 #' @param source_map_contents Includes the contents in the source map
 #'   information.
 #' @param omit_source_map_url Disable the inclusion of source map information in
-#'   the output file. Note: must specify \code{output_path} when \code{TRUE}.
+#'   the output file. Note: must specify `output_path` when `TRUE`.
 #'
 #' @return List of Sass compiler options to be used with
-#'   \code{\link{sass}}.
+#'   [sass()].
 #'
 #' @examples
 #' sass(
```

Comment:
It would be good to have examples showing how `sass_options_get` and `sass_options_set` work. The overlaying behavior could be a bit confusing for users.